### PR TITLE
Add TCP stats to the Linkerd Pod Grafana dashboard (#2329)

### LINKED ISSUE #2329: Add TCP stats to the Linkerd Pod Grafana dashboard
We should expose TCP stats in the Pod dashboard. When this happens, we should also link to this dashboard from the TCP tables in the dashboard. (https://github.com/linkerd/linkerd2/pull/2262#discussion_r)

### PR DESCRIPTION
**Pull request by liquidslr**
Fixes #2329

Signed-off-by: Gaurav Kumar <gaurav.kumar9825@gmail.com>

cc @rmars

### CODE REVIEW

File: grafana/dashboards/pod.json [hunk 1]
```diff
@@ -15,8 +15,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": null,
-  "iteration": 1531434418636,
+  "iteration": 1552075566256,
```
Comment (klingerf) [inline grafana/dashboards/pod.json]: Can I trouble you to manually revert the change to the "id" and "iteration" fields here?

Comment (liquidslr) [reply grafana/dashboards/pod.json]: sure will do that

**Comment by liquidslr** [comment]
@klingerf  minimized the stats and added the link in the web ui, anything else that needs to be done?

**Comment by liquidslr** [comment]
Didn't realize that extra rows have to be added to fix the above issue, have added them now, please have a look.
